apps/marketplace/src/app/item/[songId]/page.tsx

```diff
@@ -86,6 +92,93 @@ const SingleSong: FunctionComponent<SingleSongProps> = ({ params }) => {
                 { songData?.artist.firstName } { songData?.artist.lastName }
               </Typography>
             </Stack>
+
+            <Formik
+              initialValues={ { streamTokens: 1 } }
+              onSubmit={ () => {
+                return;
+              } }
+            >
+              { () => {
+                return (
+                  <Form>
+                    <Stack gap={ 2.5 } mb={ 4 } mt={ 0.5 }>
+                      <Stack>
+                        <Box alignSelf="flex-end" p={ 0.5 }>
+                          <Tooltip
+                            title={
+                              "The number of Stream Tokens correlates directly " +
+                              "with the percentage of Streaming royalties you " +
+                              "can acquire and the total price of the bundle. " +
+                              "For example 1 token is worth = 0.0000001% of " +
+                              "total royalties, and costs ‘Ɲ3.0‘."
+                            }
+                          >
+                            <IconButton sx={ { padding: 0 } }>
+                              <InfoIcon sx={ { color: theme.colors.grey100 } } />
```

Comment:
NEWM Studio uses a question mark icon consistently, per Marketplace Figma a "i" info icon is used. I left a comment in Figma for design to review, if alignment is needed we will update in future PR.